lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -110,6 +110,8 @@ import Data.Coerce
     ( coerce )
 import Data.Either
     ( isRight )
+import Data.Either.Combinators
+    ( fromRight' )
```

Comment:
Why not:

https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Either.html#v:fromRight ?

Reply:
used fromRight